src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -16,6 +16,7 @@ See the LICENSE file in the project root for more information.
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CppCompilerAndLinker Condition="'$(CppCompilerAndLinker)' == '' and '$(TargetOS)' == 'OSX'">clang</CppCompilerAndLinker>
+    <CppCompilerAndLinker Condition="'$(CppCompilerAndLinker)' == '' and '$(TargetOS)' == 'Linux' and '$(__ClangMajorVersion)' != '' and $(__ClangMinorVersion) != ''">clang-$(__ClangMajorVersion).$(__ClangMinorVersion)</CppCompilerAndLinker>
```

Comment:
Could you please set CppCompilerAndLinker in buildvars-setup.sh instead?

The msbuild files under src/BuildIntegration are included in the ILCompiler NuGet package. They are used in dotnet cli integration. buildvars-setup.sh is for building CoreRT itself.